conf/routes

```diff
@@ -199,8 +199,8 @@ GET       /api/assembly/:aid/contribution/:coid/feedback
 GET       /api/assembly/:aid/group/:gid/contribution/:coid/feedback              controllers.Contributions.readContributionFeedbackPrivate(aid: Long, gid: Long, coid: Long, type: String ?= null)
 GET       /api/contributions                                                     controllers.Contributions.all(page?="1", by_uuid?="", by_theme_id?="")
 
-GET       /api/assembly/:aid/campaign/:cid/contribution/:coid/body               controllers.Contributions.findAssemblyContributionPadBody(aid: Long, cid: Long, coid: Long, rev: Integer ?= null, format: String ?= null)
-GET       /api/contribution/:coid/body                                           controllers.Contributions.findContributionPadBody(coid: Long, rev: Integer ?= null, format: String ?= null)
+GET       /api/assembly/:aid/campaign/:cid/contribution/:coid/body               controllers.Contributions.findAssemblyContributionPadBody(aid: Long, cid: Long, coid: Long, rev: Long, format: String ?= null)
+GET       /api/contribution/:coid/body                                           controllers.Contributions.findContributionPadBody(coid: Long, rev: Long, format: String ?= null)
```

Comment:
My mistake. This should have been the UUID of the contribution, not its long id.